pkg/gardenlet/controller/federatedseed/shootstate/common_sync.go

```diff
@@ -112,6 +115,25 @@ func (s *SyncController) createShootStateSyncReconcileFunc(ctx context.Context,
 	}
 }
 
+func shouldSkipExtensionObjectSync(extensionObject extensionsv1alpha1.Object, kind string, req *reconcile.Request, log *logrus.Entry) bool {
+	if extensionObject.GetDeletionTimestamp() != nil {
+		return true
+	}
+
+	annotations := extensionObject.GetAnnotations()
+	operationAnnotation, ok := annotations[constants.GardenerOperation]
+	lastOperation := extensionObject.GetExtensionStatus().GetLastOperation()
+	if (lastOperation == nil || lastOperation.GetType() == v1beta1.LastOperationTypeCreate) && !ok {
+		log.Infof("Skipping ShootState sync because resoruce with kind %s in namespace %s has no operation annotation %s", kind, req.NamespacedName)
```

Comment:
```suggestion
		log.Infof("Skipping ShootState sync because resource with kind %s in namespace %s has no operation annotation %s", kind, req.NamespacedName)
```